scss/_pagination.scss

```diff
@@ -75,7 +75,7 @@
     margin-left: $pagination-margin-start;
   }
 
-  @if $pagination-margin-start == ($pagination-border-width * -1) {
+  @if $pagination-margin-start == calc($pagination-border-width * -1) {
```

Comment:
Except if you've found a non-working use case, this modification should be reverted (see https://github.com/twbs/bootstrap/commit/a153f360eea855f29947ac14be14e29380ddebeb).

Reply:
For such a small change (which is repeated a few times I'll admit), I think this is okay to leave in as it's motivated by the other changes here.